Options.py

```diff
@@ -126,6 +126,23 @@ class Option(typing.Generic[T], metaclass=AssembleOptions):
     # can be weighted between selections
     supports_weighting = True
 
+    plain_text_doc = False
```

Comment:
Rather than attach this to the `Option` class itself, wouldn't it make more sense to make this an attribute on `WebWorld` itself that applies to all options?

I don't see a reason why anyone migrating to reST formatting wouldn't just update all their doccomments, and depending on which becomes default (plain text should be default imo), if a dev decides to use the other, they'll have to set it on every option class.